docs/pages/enroll-resources/workload-identity/introduction.mdx

```diff
@@ -1,157 +1,106 @@
-host as the workloads and is accessed using a unix socket rather than a TCP
-endpoint. It can perform basic authentication and authorization of the workload
-before issuing SVIDs. This is known as Workload Attestation.
-
-## Teleport's Workload Identity
-
-Teleport's Workload Identity is an implementation of SPIFFE. Each Teleport
-cluster acts as a SPIFFE trust domain, with the Auth Service as a certificate
-authority for issuing SVIDs.
-
-Teleport's RBAC system is used to control which Bots and Users are able to
-request a SVID for a given SPIFFE ID. Roles can specify which SPIFFE IDs can
-be issued and this role is then granted to the Bot or User. For example:
-
-```yaml
-kind: role
-version: v6
-metadata:
-  name: europe-foo-svid-issuer
-spec:
-  allow:
-    spiffe:
-    - path: "/region/europe/svc/foo"
-```
-
-The SPIFFE Workload API is implemented as a configurable service within the
-`tbot` agent. The `tbot` agent should be installed close to the workloads that
-need to request SVIDs, and they can then use the Workload API exposed by `tbot`
-to fetch SVIDs and Trust Bundles.
-
-Teleport's Workload Identity supports issuing X.509-SVIDs and JWT-SVIDs.
+(!/docs/pages/includes/workload-id-preview.mdx!)
+
+Teleport Workload Identity securely issues short-lived cryptographic identities
+to workloads. It is a flexible foundation for workload identity across your 
+infrastructure, creating a uniform way for your workloads to authenticate 
+regardless of where they are running.
+
+The flexible nature of Teleport Workload Identity enables it to be used for a
+range of purposes, including:
+
+- Workload authentication to third-party APIs on cloud platforms such as AWS, 
+  GCP and Azure.
+- Providing X.509 certificates for mutual TLS authentication between workloads
+  within your infrastructure as part of a Zero Trust strategy.
+- Workload authentication between services within your infrastructure. 
+
+Teleport Workload Identity is compatible with the open-source
+[Secure Production Identity Framework For Everyone (SPIFFE)](./spiffe.mdx)
+standard. This enables interoperability between workload identity
+implementations and also provides a wealth of off-the-shelf tools and SDKs to
+simplify integration with your workloads. 
+
+There's a whole host of benefits to adopting Teleport Workload Identity, but
+here's some of the key ones:
+
+- Eliminate the use of long-lived shared secrets within your infrastructure, and
+  reduce the risk of exfiltration and time spent by engineers creating and 
+  rotating these secrets.
+- Establish an out of the box universal form of identity for your workloads,
+  enabling your engineers to get on with building new services without needing 
+  to think about how they'll authenticate.
+- Converge on a first-class form of identity for your workloads, simplifying
+  infrastructure by reducing the number of different ways workloads authenticate.
```

Comment:
I would shorten this introductory section by combining the list of benefits with the list of use cases.